components/dashboard/src/components/Menu.tsx

```diff
@@ -4,83 +4,191 @@
+            <div className="flex">
+                <div className="flex justify-between items-center pr-3">
+                    <Link to="/">
+                        <img src={gitpodIcon} className="h-6" />
+                    </Link>
+                    <div className="ml-2 text-base">
+                        {showTeamsUI
+                            ? <ContextMenu classes="w-64 left-0" menuEntries={[
+                                {
+                                    title: userFullName,
+                                    customContent: <div className="w-full text-gray-400 flex flex-col">
+                                        <span className="text-gray-800 text-base font-semibold">{userFullName}</span>
+                                        <span className="">Personal Account</span>
+                                    </div>,
+                                    separator: true,
+                                    onClick: () => history.push("/"),
+                                },
+                                ...(teams || []).map(t => ({
+                                    title: t.name,
+                                    customContent: <div className="w-full text-gray-400 flex flex-col">
+                                        <span className="text-gray-800 text-base font-semibold">{t.name}</span>
+                                        <span className="">{!!teamMembers[t.id]
+                                            ? `${teamMembers[t.id].length} member${teamMembers[t.id].length === 1 ? '' : 's'}`
+                                            : '...'
+                                        }</span>
+                                    </div>,
+                                    separator: true,
+                                    onClick: () => history.push(`/${t.slug}`),
+                                })).sort((a,b) => a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1),
+                                {
+                                    title: 'Create a new team',
+                                    customContent: <div className="w-full text-gray-400 flex items-center">
+                                        <span className="flex-1 font-semibold">New Team</span>
+                                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 14 14" className="w-3.5"><path fill="currentColor" fill-rule="evenodd" d="M7 0a1 1 0 011 1v5h5a1 1 0 110 2H8v5a1 1 0 11-2 0V8H1a1 1 0 010-2h5V1a1 1 0 011-1z" clip-rule="evenodd"/></svg>
+                                    </div>,
+                                    onClick: () => history.push("/new-team"),
+                                }
+                            ]}>
+                                <div className="flex p-1.5 pl-3 rounded-lg hover:bg-gray-200">
+                                    <span className="text-base text-gray-600 font-semibold">{team?.name || userFullName}</span>
```

Comment:
**issue**: Reminder to restore the colors for dark theme! These can use the colors of the menu items on the right. Font size can be `text-sm`! 🌙

Reply:
@gtsiolis Unless I'm looking at the wrong spec, the [Figma file](https://www.figma.com/file/zU2ZmLwZzegEAHKwRH3xkL/Team-Members-%234421?node-id=0%3A1) currently has `text-base` (i.e. `16px`) for team member names, not `text-sm` (i.e. `14px`).

Reply:
@jankeromnes You are absolutely right! The initial specs are using `16px` for the main and secondary navigation on the top right which is now still using a `14px` size. May I suggest the following two items:
1. Either use the smaller size of align all navigation elements and make sure they are using a `32px` height?
2. If we go with the specs (`16px`), semibold here is rendering quite bold so I'd either change the font weight to medium or even better enable font-smoothing for the product as in https://github.com/gitpod-io/website/pull/222.

Reply:
Now tracked in https://github.com/gitpod-io/gitpod/issues/4471